numpyro/distributions/util.py

```diff
@@ -453,6 +441,12 @@ def signed_stick_breaking_tril(t):
     return y
 
 
+def logsumexp(x, axis=0, keepdims=False):
+    x_max = np.max(x, axis=axis, keepdims=True)
```

Comment:
Nice! Just curious - is the upstream `logsumexp` implementation doing something else that is causing issues with fastmath mode?

Reply:
I think this is the only issue of NumPyro with JAX which have not been isolated yet. It is related to fastmath mode. I will ping you when I have more information. Here is the [upstream implementation](https://github.com/google/jax/blob/master/jax/scipy/special.py#L77) which does something else with -inf...

Reply:
Ahh..interesting. Feel free to create a NumPyro issue with your comment, for future reference.

Reply:
Actually, we have a TODO in neutra example which points to a similar issue.